Automattic/wp-calypso
# Media: Reduxify media library selected items store

This PR reduxifies the media library selected items store. Since it's a bit too big, I'm splitting it into smaller, more focused PRs:

* #40271
* #40272
* #40273
* #40274

This PR can be used to see the big picture and test the entire implementation.

See #39379 for the full media stores reduxification effort. 

Also see #39372 for a similar effort that reduxified the media validation store.

#### Changes proposed in this Pull Request

* Reduxify the media library selected items store

#### Testing instructions

* Follow the testing instructions of each of the above PRs.
* Use this PR to smoke test the media library and media management and upload functionality in the post editor and site settings.
* Everything should behave as it did before.
